config/updates.jenkins.io.yaml

```diff
@@ -5,12 +5,13 @@ global:
     annotations:
       "cert-manager.io/cluster-issuer": "letsencrypt-prod"
       "nginx.ingress.kubernetes.io/ssl-redirect": "true"
+      "nginx.ingress.kubernetes.io/use-regex": "true" # Required to allow regexp path matching with Nginx
     hosts:
       - host: azure.updates.jenkins.io
         paths:
           - path: /
             backendService: httpd
-          - path: /.*[.](deb|hpi|war|rpm|msi|pkg|sha256|md5sum|zip|gz|pdf|json|svg|sh|jpeg|ico|png|html)$
+          - path: /.*[.](deb|hpi|war|rpm|msi|pkg|sha256|md5sum|zip|gz|pdf|json|svg|sh|jpeg|ico|png|html|txt)$ # Requires the regexp engine of Nginx to be enabled
```

Comment:
`txt` is added only for our testing isn't it? Should we plan to remove it later?

Reply:
Absolutely, this was for tests (and healthchecking mirrors).

Added a new "TODO" in https://github.com/jenkins-infra/helpdesk/issues/2649#issuecomment-1784873731 to ensure this pattern list is revised before production